packages/inter-protocol/src/proposals/replaceElectorate.js

```diff
@@ -137,6 +138,45 @@ const inviteECMembers = async (
   await distributeInvitations(zip(values(voterAddresses), invitations));
 };
 
+/**
+ * Invites members to the Economic Charter by creating and distributing charter
+ * member invitations to the specified voter addresses.
+ *
+ * @param {EconomyBootstrapPowers} powers - The bootstrap powers required for
+ *   economic operations, including `namesByAddressAdmin` used for managing
+ *   names and deposits.
+ * @param {{
+ *   options: {
+ *     voterAddresses: Record<string, string>;
+ *     econCharterKit: {
+ *       creatorFacet: {
```

Comment:
IIUC, the config is plain data. How are we getting an object with a method in there?

Reply:
oh... this isn't an exported function referred to by a behavior.

Still.. the name `config` threw me off. Maybe `opts` instead?